ansible_events_ui/api.py

```diff
@@ -507,6 +509,52 @@ async def list_job_instances(db: AsyncSession = Depends(get_db_session)):
     return result.all()
 
 
+@router.post("/api/job_instance/")
+async def create_job_instance(
+    j: JobInstance, db: AsyncSession = Depends(get_db_session)
+):
+
+    query = select(playbooks).where(playbooks.c.id == j.playbook_id)
+    playbook_row = (await db.execute(query)).first()
+
+    query = select(inventories).where(inventories.c.id == j.inventory_id)
+    inventory_row = (await db.execute(query)).first()
+
+    query = select(extra_vars).where(extra_vars.c.id == j.extra_var_id)
+    extra_var_row = (await db.execute(query)).first()
+
+    job_uuid = str(uuid.uuid4())
+
+    query = insert(job_instances).values(uuid=job_uuid)
+    result = await db.execute(query)
+    await db.commit()
+    (job_instance_id,) = result.inserted_primary_key
+
+    event_log = asyncio.Queue()
+
+    task = asyncio.create_task(
```

Comment:
I think we should document somewhere (if it hasn't been done already) that the tasking system implementation we use brings a limitation on number of worker processes that are used to run the application. The application is limited to only a single process mode and will have unexpected side effects if run in multi process mode.